.gitignore

```diff
@@ -1,3 +1,4 @@
 target
 Cargo.lock
-*.rs.bk
\ No newline at end of file
+*.rs.bk
+.idea
```

Comment:
Please add editor specific ignore lines to your global `.gitignore`, not to a project specific `.gitignore`.

Reply:
Removed the gitignore pattern here. I'm not sold on global .gitignore since I would commit most of the .idea directory in projects where it's not objected to.